studio/components/interfaces/Organization/BillingSettingsV2/Subscription/PlanUpdateSidePanel.tsx

```diff
@@ -73,6 +73,9 @@ const PlanUpdateSidePanel = () => {
     }
   )
 
+  const billingViaPartner = subscription?.billing_via_partner === true
```

Comment:
im guessing this requires an API change as well? but still safe to go through i guess with the `?`